app/models/user.rb

```diff
@@ -297,6 +297,7 @@ def admin?
     has_role?(:super_admin)
   end
 
+  # this method will be optimized in a seperate PR
```

Comment:
You can kill this too if you want, I opened up an optimization PR that way you can keep the scope of this nice and narrow :)

Reply:
awesome - thanks Molly! ❤️